devel/logcat-colorize/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+name                logcat-colorize
+version             0.8.1
```

Comment:
These 2 lines will be redundant and should be removed when you use the GitHub portgroup.